manifests/components/oauth2-proxy.jsonnet

```diff
@@ -44,13 +44,36 @@ local OAUTH2_PROXY_IMAGE = (import "images.json")["oauth2_proxy"];
 
   hpa: kube.HorizontalPodAutoscaler($.p + "oauth2-proxy") + $.metadata {
     target: $.deploy,
-    spec+: {maxReplicas: 10},
+    spec+: {
+      // Put a cap on growth due to (eg) DoS attacks.
+      // Large sites will want to increase this to cover legitimate demand.
+      maxReplicas: 10,
```

Comment:
Do we need a minReplicas: 2 or some such to prevent auto-scaling from messing things up?

Reply:
ditto